test/e2e/suite_test.go

```diff
@@ -93,11 +79,29 @@ var _ = BeforeSuite(func() {
 	k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})
 	Expect(err).NotTo(HaveOccurred())
 	Expect(k8sClient).NotTo(BeNil())
+
+	LwsReadyForTesting(k8sClient)
 })
 
 var _ = AfterSuite(func() {
 	cancel()
-	By("tearing down the test environment")
-	err := testEnv.Stop()
-	Expect(err).NotTo(HaveOccurred())
 })
+
+func LwsReadyForTesting(client client.Client) {
+	By("waiting for webhooks to come up")
+
+	// To verify that webhooks are ready, let's create a simple lws.
+	leaderWorkerSetSpec := testutils.BuildLeaderWorkerSet(metav1.NamespaceDefault).Replica(3).Obj()
```

Comment:
we use spec usually to define the context of a template, like podSpec for the spec of the pod template, which is a sub field of the pod template, here, I think lws may make more sense, wdyt?